src/js/components/search/FilterExpandButton.jsx

```diff
@@ -25,32 +25,20 @@ export default class FilterExpandButton extends React.Component {
             icon = <Icons.AngleDown />;
         }
 
-        let clickEvent = '';
-        switch (this.props.name) {
-            case 'Search':
-                clickEvent = null;
-                break;
-            case 'Budget Categories':
-                clickEvent = null;
-                break;
-            case 'Award ID':
-                clickEvent = null;
-                break;
-            case 'Award Amount':
-                clickEvent = null;
-                break;
-            case 'Other Award Items':
-                clickEvent = null;
-                break;
-            default:
-                clickEvent = this.props.toggleFilter;
-                break;
+        let disabledStatus = false;
+        if (this.props.name !== 'Search' ||
+            this.props.name !== 'Budget Categories' ||
+            this.props.name !== 'Award ID' ||
+            this.props.name !== 'Award Amount' ||
+            this.props.name !== 'Other Award Items') {
+                disabledStatus = true;
```

Comment:
This needs to un-indented by 4 spaces (one tab to the left) to pass the linting (getting a green checkmark in Github instead of the red X).